packages/strapi-design-system/src/hooks/useInternationalizedNumber.ts

```diff
@@ -0,0 +1,17 @@
+import { useRef } from 'react';
+
+import { NumberFormatter, NumberParser } from '@internationalized/number';
+
+import { getDefaultLocale } from '../helpers/getDefaultLocale';
+
+export const useInternationalizedNumber = (locale, { parserParams, formatParams }) => {
```

Comment:
There's no type annotation for what `locale` or the parms are, it'd also be good to add JSdoc to explain what this is used for etc.